query/stdlib/influxdata/influxdb/storage.go

```diff
@@ -151,7 +152,10 @@ type ReadWindowAggregateSpec struct {
 }
 
 // WindowAggregateCapability describes what is supported by WindowAggregateReader.
-type WindowAggregateCapability struct{}
+type WindowAggregateCapability interface {
+	reads.WindowAggregateCapability
+}
+type windowAggregateCapability struct {}
 
 // WindowAggregateReader implements the WindowAggregate capability.
 type WindowAggregateReader interface {
```

Comment:
You'll want to remove the pointer `*` in the prototype for `HasWindowAggregateCapability` below, since interfaces are usually passed by value in Go.